modules/page/html/render/settings.js

```diff
@@ -44,6 +45,23 @@ exports.create = function (api) {
                 }, name)
               })
             })
+          ]),
+          h('section', [
+            h('h2', 'Filters'),
+            computed(filterFollowing, filter => {
+              const style = filter
+                ? { 'margin-right': '1rem', 'border-color': 'teal' }
+                : { 'margin-right': '1rem' }
+
+              return [
+                h('button', {
+                  'ev-click': () => api.settings.sync.set({
+                    filters: {following: !filter}
+                  }),
+                  style
+                }, filter ? 'Hide following messages' : 'Show following messages')
+              ]
+            })
```

Comment:
This interface is a bit confusing here because it is not clear if the state on the button "Hide following messages" is the current one or what will be set if clicked. This might be better as a checkbox:

```js
h('label', [
  h('input', {
    type: 'checkbox', 
    checked: filterFollowing, 
    'ev-change': (ev) => api.settings.sync.set({
      filters: {following: ev.target.checked}
    })
  }), ' Hide following messages'
])
```

Reply:
Yes, that is a lot more clear that way.